lib/tests/detectExtension.test.js

```diff
@@ -0,0 +1,13 @@
+const { describe, it } = require('mocha');
+const { expect } = require('chai');
+const { getExtension } = require('../shared/utils');
+const { mixedSubtitles } = require('./mocks/mixedSubtitles');
+
+describe('#detectExtension', () => {
```

Comment:
Can you change the file name to `getExtension.test.js` to mimic the function?

Reply:
Makes sense, will change.